src/clib/pio_nc.c

```diff
@@ -119,13 +119,13 @@ int PIOc_inq(int ncid, int *ndimsp, int *nvarsp, int *ngattsp, int *unlimdimidp)
 
     /* A failure to inquire is not fatal */
     mpierr = MPI_Bcast(&ierr, 1, MPI_INT, ios->ioroot, ios->my_comm);
-    if(mpierr != MPI_SUCCESS){
+    if (mpierr != MPI_SUCCESS)
```

Comment:
This is a code formatting/style change and needs to go separately. IMO, it is always a good idea to add braces for statements (or statement) within an if condition.